### ISSUE TITLE
Serve static files from SD card.

### ISSUE THREAD

**Issue by athusrv** [jczic/MicroWebSrv2#47]
Hi! Is there any documentation that shows how to serve web static files (HTML, CSS, JS) from the SD card?

I am sorry if I missed something. I just started paying with this library and MicroPython.

**Comment by William04A** [comment]
Yeah, you can mount the SD card to a location like `/SD/`, and then change the RootPath for static files like so:

`mws2.RootPath = "/SD/static/" #Set the root path for static files (this location would be for the folder "static" on the SD card)`
